pkg/cmd/template/data_values_flags.go

```diff
@@ -195,3 +212,28 @@ func (s *DataValuesFlags) buildOverlay(keyPieces []string, value interface{}, de
 
 	return &yamlmeta.Document{Value: resultMap, Position: pos}
 }
+
+func (s DataValuesFlags) CreateDataValues(keyStr string, val interface{}, descStr string) (*workspace.DataValues, error) {
+	var lib string
+
+	hasLib := strings.Contains(keyStr, ":")
+	if hasLib {
+		keyParts := strings.Split(keyStr, ":")
+		lib = keyParts[0]
+		keyStr = keyParts[1]
+	}
+
+	overlayValueDoc, err := workspace.NewDataValues(s.buildOverlay(strings.Split(keyStr, "."), val, descStr))
+	if err != nil {
+		return nil, err
+	}
+
+	if hasLib {
+		err = overlayValueDoc.SetLibAndTag(lib)
```

Comment:
would be better to have a constructor that takes doc and lib instead of having public `SetLibAndTag` that we set later == makes DataValues more immutable.